tls/s2n_kex.c

```diff
@@ -105,17 +110,49 @@ static int s2n_get_server_hybrid_extensions_size(const struct s2n_connection *co
     return s2n_kex_server_extension_size(hybrid_kex_0, conn) + s2n_kex_server_extension_size(hybrid_kex_1, conn);
 }
 
-static const struct s2n_kex s2n_sike = {
+static int s2n_setup_basic_kex(struct s2n_connection *conn)
```

Comment:
If this function is a no-op, I don't think we should use this name. Maybe `s2n_setup_empty_kex` or `s2n_no_op_kex`?